src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.js

```diff
@@ -17,8 +19,8 @@
         liveNotebook = true;
     }
     catch (err) {
-        // log the error, just in case we *are* in a live notebook
-        console.log('[toc2] working in non-live notebook:', err);
+        // We *are* in a live notebook
```

Comment:
It was a typo. I have removed the print of the err because it is printed in the non-livenotebook version, as it should, but this may suggest to the user that there is a problem..